sw/device/lib/testing/aon_timer_testutils.c

```diff
@@ -0,0 +1,35 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+#include "sw/device/lib/testing/aon_timer_testutils.h"
+
+#include <stdbool.h>
+#include <stdint.h>
+
+#include "sw/device/lib/dif/dif_aon_timer.h"
+#include "sw/device/lib/testing/check.h"
+
+#include "hw/top_earlgrey/sw/autogen/top_earlgrey.h"
+
+void aon_timer_wakeup_config(uint32_t cycles) {
+  dif_aon_timer_t aon_timer;
+  dif_aon_timer_params_t params = {
+      .base_addr = mmio_region_from_addr(TOP_EARLGREY_AON_TIMER_AON_BASE_ADDR),
+  };
+  CHECK(dif_aon_timer_init(params, &aon_timer) == kDifAonTimerOk);
```

Comment:
IMO let the top level test create the context and pass it as a handle so that this can be used across other top level implementations where the block may live in a different address space.

Reply:
@timothytrippel, does this make sense? Or do you have other recommendations?

Reply:
Done